iron-doc-viewer.html

```diff
@@ -119,6 +119,22 @@
         observer: '_descriptorChanged',
       },
 
+      /**
+       * Set to true when running in the catalog.
+       *
+       * By default, hash link includes the component name,
+       * so links work when an `iron-component-page` is
+       * served statically. In the catalog, the component name
+       * is omitted because it's already included in a URL parameter.
+       *
+       * In the catalog, the app handles routing &
+       * scrolling to the selected anchor.
+       */
+      catalog: {
```

Comment:
Proposal: we can have a new property:
```js
anchorFormat: {
  type: String,
  value: '%prefix%type%suffix'
}
```
so that in line 285 you can do:  `return this.anchorFormat.replace('%prefix', prefix).replace('%type', type);` and so on... 
From iron-component-page, you can change `anchorFormat` to `%type%suffix` based on the catalog property: https://github.com/PolymerElements/iron-component-page/blob/master/iron-component-page.html#L372

wdyt?